step2/hackatom-stash-controller.sh

```diff
@@ -1,79 +1,106 @@
 #!/usr/bin/env bash
 set -e
 
-# This script is for setting up the stash and controller accounts.
+# This script is for setting up the stash and controller accounts, and sending
+# some txs from controller on behalf of stash. All commands are commented.
 
 # Set variables
 CFG_DIR=~/.simapp
-BUILD_CMD=./cosmos-sdk/build/simd
+BUILD_CMD=../step1/cosmos-sdk/build/simd
 VALIDATOR=alice
 CHAIN_ID=hackatom6-chain
+MIN_FEES=20stake
 
 # Key names in keyring
 STASH=stash
 CONTROLLER=controller
 
 # Get validator address
-validator=$("${BUILD_CMD}" keys show $VALIDATOR --bech val --keyring-backend test --output json)
-VALIDATOR_ADDRESS=$(echo "${validator}" | jq -r '.address')
+VALIDATOR_ADDRESS=$($BUILD_CMD keys show $VALIDATOR --bech val -a --keyring-backend test)
 
-# Create stash account in keyring.
+echo
+echo "Create stash account in keyring."
```

Comment:
So I changed all `#` into `echo`, do you think that's okay @likhita-809?

I think it helps people running this script all at once.

Reply:
yeah, seems okay